docs/single_member_restoration.md

```diff
@@ -0,0 +1,15 @@
+## Restoration of a single member in multi-node etcd-backup-restore cluster.
```

Comment:
```suggestion
# Restoration of a single member in multi-node etcd cluster.
```

Reply:
hmm .... Isn't mentioning only `multi-node etcd` might be ambiguous as we want to restore `multi-node etcd` but it's possible only when backup-restore should be running as sidecar. That's why I mentioned `multi-node etcd-backup-restore`. Although I don’t have any hard opinion on this. I'm open for change.

Reply:
Technically, it's an `etcd` cluster and there is no such thing as a `etcd-backup-restore` cluster. I think that's pretty clear if one uses Druid that a backup-restore side-car is involved.
However, I get your point and if you'd like to emphasize this then I propose to just mentioned it in an additional sentence.

Reply:
Updated. PTAL